src/core/evaluator.js

```diff
@@ -1997,23 +1998,39 @@ class PartialEvaluator {
       width: 0,
       height: 0,
       vertical: false,
+      lastCharSize: 0,
       lastAdvanceWidth: 0,
       lastAdvanceHeight: 0,
       textAdvanceScale: 0,
       spaceWidth: 0,
-      fakeSpaceMin: Infinity,
-      fakeMultiSpaceMin: Infinity,
-      fakeMultiSpaceMax: -0,
-      textRunBreakAllowed: false,
+      spaceInFlowMin: 0,
+      spaceInFlowMax: 0,
+      kerningSpaceMin: Infinity,
```

Comment:
Doesn't this need to be `trackingSpaceMin` instead, given the latest changes below?